### PR TITLE
Some tweaks to the Custom Components Guide

### PR DESCRIPTION
**Pull request by abidlabs**
As I was building a custom component, noticed some small areas of improvement for the custom components guides:

* Removed the old "Create a New Component" Guide since that was the first thing that showed up when I googled Gradio Custom Component, and made the old URL redirect to the new Guide
* Various nits

### CODE REVIEW

File: guides/05_custom-components/02_key-component-concepts.md [hunk 1]
```diff
@@ -39,16 +39,15 @@ with gr.Blocks() as demo:
 demo.launch()
 ```
 
-The interactive version of the component is much more complex -- you can upload and edit images, draw sketches, etc. -- while the static version does not do any of that.
+The interactive version of the component is much more complex -- you can upload images or snap a picture from your webcam -- while the static version can only be used to display images.
 
-Not every component has an interactive version.
-For example, the `gr.AnnotatedImage` only appears as a static version since there's no way to interactively change the value of the annotations or the image.
+Not every component has a distinct interactive version. For example, the `gr.AnnotatedImage` only appears as a static version since there's no way to interactively change the value of the annotations or the image.
```
Comment (freddyaboulton) [inline guides/05_custom-components/02_key-component-concepts.md:44]: This and the current source are equivalent.

File: js/_website/src/routes/redirects.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export const redirects = {
+	"/guides/creating-a-new-component": "/guides/five-minute-guide",
```
Comment (freddyaboulton) [inline js/_website/src/routes/redirects.js:2]: 👍

**Review by freddyaboulton — APPROVED** [review]
Thanks @abidlabs !

**Comment by abidlabs** [comment]
Cool, working on the BlurHashGallery custom component just fyi

**Comment by freddyaboulton** [comment]
The what? 👀

**Pull request merged**